pkg/pillar/hypervisor/kvm_test.go

```diff
@@ -2819,11 +2828,13 @@ func expectedMultifunctionDevice() string {
   host = "00:0b.0"
   bus = "pci.2"
   addr = "0x0"
+
 [device]
   driver = "vfio-pci"
   host = "00:0d.2"
   bus = "pcie-bridge.1"
-  addr = "0x2"`
+  addr = "0x2"
+`
 }
 
 func TestPCIAssignmentsTemplateFillMultifunctionDevice(t *testing.T) {
```

Comment:
I would also add tests to generate and evaluate the corresponding sections in the xen.cfg file. This will also help in understanding the exact expectations from the code.

Reply:
Added